client/app/index.js

```diff
@@ -37,3 +37,6 @@ if (!global.Intl) {
 } else {
   loadCurrentModule();
 }
+
+// Initializers
+require('./lib/helpers/confirm_dialog');
```

Comment:
@weiqingtoh This one should be required after Intl patch is applied, move this to under `require.ensure` instead.